src/sequences/SequenceConstant.ts

```diff
@@ -1,63 +1,53 @@
 import { ValidationStatus } from '@/shared/ValidationStatus';
-import { SequenceInterface, SequenceParamsSchema, SequenceExportModule } from './SequenceInterface';
+import { SequenceParamsSchema, SequenceExportModule } from './SequenceInterface';
+import { SequenceClassDefault } from './SequenceClassDefault';
+
 /**
  *
  * @class SequenceConstant
- * extends the sequenceClassDefault, by setting params and some custom implementation
- * for the generator and the fillCache function.
+ * Extends the sequenceClassDefault, by changing the parameter schema
+ * and reimplementing the getElement function.
  */
-class SequenceConstant implements SequenceInterface{
-    ID: number;
+class SequenceConstant extends SequenceClassDefault {
     name = "Constant Sequence";
     description = "A sequence that is constant";
-    params: SequenceParamsSchema[] = [new SequenceParamsSchema(
+    params = [new SequenceParamsSchema(
         'constantValue',
         'number',
         'Constant Value',
         false,
         '0'
     )];
-    private settings: { [key: string]: string|number|boolean} = {};
-    private requested = 0;
-    private ready = false;
-    private finite = false;
-    private isValid = false;
-
-    constructor(ID: number, finite?: boolean) {
-        this.ID = ID;
-        this.finite = finite || true;
-        this.ready = false;
-    }
+    value = -1;
 
-    initialize(){
-        if(this.isValid){
-            this.ready = true;
-            return;
-        } else {
-            throw "Sequence is not valid. Run validate() and address any errors."
-        }
+    constructor(ID: number) {
+        super(ID);
     }
 
     validate(): ValidationStatus{
-		this.params.forEach(param => {
-            this.settings[param.name] = param.value;
-		});
+        this.settings['name'] = 'Constant';
+	const svs = super.validate();
```

Comment:
Is `svs` a common abbreviation? I generally try to avoid short variables unless they're really common like `req` for request or `e` for error.

Reply:
It was short for "super validation status" but your feedback is excellent. I will fix in a further commit shortly.